app/views/user_mailer/project_complete.html.slim

```diff
@@ -0,0 +1,51 @@
+p Hello #{@owner.display_name},
+
+p Wow! Your project is 100% transcribed!  What's next?
+ul
+  li Send a wrap up email to all of your collaborators announcing they did it! Remind them of the purpose of the project, thank them for their work, and direct them on to your next project or a colleague's project.
+  br
+  li 
+    'Export your transcriptions if you'd like. You can export your transcriptions in a variety of formats to use for other purposes, 
+    =link_to "here", collection_export_path(@collection.owner, @collection, only_path: false)
+    |. Or you can use the IIIF API.  If you need custom integration services to pull them into a downstream system, we can help - send us an email at 
+    =mail_to 'support@fromthepage.com'
+    |.
+  br
+  li You can also leave your project up on FromThePage - it has great Google SEO. Our perspective is that a transcription is never done - someone may read something years from now and be able to correct a confusing point or add more context to your project.
```

Comment:
Let's make this 2 sentences:

Our perspective is that a transcription is never done - someone

Our perspective is that a transcription is never done.  Someone